pkg/manager/options.go

```diff
@@ -152,3 +156,17 @@ func (o *Options) defaults() {
 		o.PodNamespace = DefaultPodNamespace
 	}
 }
+
+func (o *Options) getCredentials() map[string]string {
```

Comment:
I would have this return an `error` too instead of calling `os.Exit` from within